### PR TITLE
test $

### PR DESCRIPTION
**Pull request by sigmaSd**
To get 100 % coverage

image

to

image

### CODE REVIEW

File: command_builder.test.ts [hunk 1]
```diff
@@ -13,7 +13,8 @@ Deno.test("Quick example", async () => {
 });
 
 Deno.test("xargs one-line", async () => {
-  const result = await $`echo hello`
+  const result = await $`echo ignore`
+    .$(`echo hello`)
```
Comment (impactaky) [inline command_builder.test.ts]: This is not xargs test.
Can you separate to other test?

**Review by impactaky — COMMENTED** [review]
Thank you!

File: command_builder.test.ts (hunk 1 above)
Comment (sigmaSd) [reply command_builder.test.ts]: done

**Comment by impactaky** [comment]
Thank you!

**Pull request merged**